tests/hikari/impl/test_rest.py

```diff
@@ -4815,7 +4821,39 @@ async def test_create_slash_command(self, rest_client: rest.RESTClientImpl):
             rest_client._request.return_value, guild_id=None
         )
         rest_client._request.assert_awaited_once_with(
-            expected_route, json={"type": 1, "name": "okokok", "description": "not ok anymore"}
+            expected_route,
+            json={
+                "type": 1,
+                "name": "okokok",
+                "description": "not ok anymore",
+            },
+        )
+
+    async def test_create_slash_command_with_name_and_description_localizations(self, rest_client: rest.RESTClientImpl):
+        expected_route = routes.POST_APPLICATION_COMMAND.compile(application=4332123)
+        rest_client._request = mock.AsyncMock(return_value={"id": "29393939"})
+
+        result = await rest_client.create_slash_command(
+            StubModel(4332123),
+            "okokok",
+            "not ok anymore",
+            name_localizations={locales.Locale.TR: "pırt"},
+            description_localizations={locales.Locale.TR: "jej"},
+        )
+
+        assert result is rest_client._entity_factory.deserialize_slash_command.return_value
+        rest_client._entity_factory.deserialize_slash_command.assert_called_once_with(
+            rest_client._request.return_value, guild_id=None
+        )
+        rest_client._request.assert_awaited_once_with(
+            expected_route,
+            json={
+                "type": 1,
+                "name": "okokok",
+                "description": "not ok anymore",
+                "name_localizations": {locales.Locale.TR: "pırt"},
+                "description_localizations": {locales.Locale.TR: "jej"},
+            },
```

Comment:
Instead of adding a test case for this, edit the current one to also pass in the new args